src/khoj/configure.py

```diff
@@ -303,9 +304,16 @@ def configure_middleware(app):
     app.add_middleware(SessionMiddleware, secret_key=os.environ.get("KHOJ_DJANGO_SECRET_KEY", "!secret"))
 
 
-@schedule.repeat(schedule.every(22).to(26).hours)
+@schedule.repeat(schedule.every(22).to(25).hours)
 def update_search_index():
     try:
+        if ProcessLockAdapters.is_process_locked(ProcessLock.Operation.UPDATE_EMBEDDINGS):
+            logger.info("🔒 Skipping update search index due to lock")
```

Comment:
nit: Let's consistently use the term "content index" instead of "search index"
```suggestion
            logger.info("🔒 Skipping update content index due to lock")
```